editor-components/content/content4/content4.module.scss

```diff
@@ -1,177 +1,205 @@
+
+    .bottom {
+      display: flex;
+      gap: $gap;
+      flex: 2;
+
+      .item {
+        flex: 1;
+      }
+    }
+  }
+
+  .item {
+    padding: 25px;
+    display: flex;
+    justify-content: flex-start;
+    position: relative;
+    height: 100%;
+    width: 100%;
+
+    .background-image {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      overflow: hidden;
+
+      img {
+        position: absolute;
+        left: -10px;
+        transition: left 0.3s ease;
+        width: calc(100% + 20px);
+        height: 100%;
+        object-fit: cover;
+      }
+    }
+
+    &:hover .background-image img {
+      left: 0;
+    }
+
+    .content {
+      z-index: 5;
+      display: flex;
+      color: white;
+      max-width: 90%;
+      margin-top: auto;
+      margin-bottom: 10%;
+      height: auto;
+      font-size: 1rem;
+      white-space: normal;
+
+      .title {
+        font-size: 2.3rem;
+        font-weight: bold;
+        color: white;
+        overflow: hidden;
+        display: -webkit-box;
+        -webkit-line-clamp: 2;
+        -webkit-box-orient: vertical;
+        margin-right: 20px;
+      }
+
+      .icons {
+        font-size: 4rem;
+        display: flex;
+        align-items: center;
+        margin-left: 0;
+        transition: margin-right 0.3s ease;
+        position: relative;
+        width: 100px;
+      }
+    }
+    .icon1,
+    .icon2 {
+      position: absolute;
+    }
+    .icon1 {
+      left: 0;
     }
+    .icon2 {
+      left: 40px;
+    }
+    &:hover .icons .icon2 {
+      left: 50px;
+    }
+  }
 }
 
-@media only screen and (max-width: 768px) {
-    .container {
-        padding: 50px 20px;
-        .max-content {
-            align-items: center;
-            flex-direction: column;
-            .card-item-count {
-                min-width: 100%;
-            }
-            .content {
-                .horizontal {
-                    min-width: 320px;
-                }
-            }
-        }
-        .small-image-container {
-            display: flex;
-            margin-top: 30px;
-            flex-wrap: wrap;
-            .small-image {
-                display: flex;
-                align-items: center;
-                text-align: center;
-                padding: 10px;
-                margin-top: 20px;
-                img {
-                    margin: 5px;
-                    max-width: 25px;
-                }
-            }
+@media only screen and (max-width: 1170px) {
+  .container {
+    padding-bottom: 1%;
+
+    .max-content {
+      display: grid;
+      grid-template-rows: 1fr 1.25fr;
+      height: auto;
+    }
+
+    .left {
+      grid-row: 1 / 2;
+      .item {
+        align-items: flex-start;
+        padding: 50px;
+        height: 100%;
+      }
+    }
+
+    .right {
+      grid-row: 2 / 3;
+      display: flex;
+      gap: 1%;
+      height: 100%;
+    }
+    .item {
+      .content {
+        margin-top: 75%;
```

Comment:
why is there a margin top 75%? please check

Reply:
checked. it's necessary